Tasks/DownloadPackageV0/package.json

```diff
@@ -17,11 +17,14 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@types/node": "^6.14.2",
-    "azure-devops-node-api": "9.0.1",
-    "azure-pipelines-task-lib": "2.8.0",
+    "@types/node": "^10.17.0",
+    "azure-devops-node-api": "10.1.2",
+    "azure-pipelines-task-lib": "3.0.6-preview.0",
     "decompress-zip": "0.3.0",
-    "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz",
-    "utility-common": "file:../../_build/Tasks/Common/utility-common-1.0.2.tgz"
+    "azure-pipelines-tasks-packaging-common": "2.0.0-preview.0",
```

Comment:
Could you please push package.lock.json changes also?

Reply:
Updated